From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Developer <developer@eclipse.org>
Date: Mon, 6 Jan 2026 12:00:00 +0000
Subject: [PATCH] Export interprocess_notification target for external use

The interprocess_notification target in score/os/utils/interprocess is
currently restricted to internal test packages only. This patch makes it
available to external consumers who depend on score_baselibs.

This is required by score_communication tests that need inter-process
synchronization primitives.
---
 score/os/utils/interprocess/BUILD | 1 +
 1 file changed, 1 insertion(+)

diff --git a/score/os/utils/interprocess/BUILD b/score/os/utils/interprocess/BUILD
index 1234567..abcdefg 100644
--- a/score/os/utils/interprocess/BUILD
+++ b/score/os/utils/interprocess/BUILD
@@ -51,6 +51,7 @@ cc_library(
     visibility = [
         "//platform/aas/mw/com/test:__subpackages__",
         "@score_baselibs//score/analysis/tracing/test:__subpackages__",
+        "//visibility:public",
     ],
     deps = [
         ":interprocess_conditional_variable",
-- 
2.40.0
